bootstrap/http_server.go

```diff
@@ -20,6 +22,9 @@ var FXModule_HTTPServer = fx.Module(
 		controllers.NewCityController,
 		repositories.NewForecastRepo,
 		controllers.NewForecastController,
+		createWeatherApiService,
+		services.NewCityService,
```

Comment:
Wrapping constructors in a function makes it a whole lot easier to pinpoint errors  when things break since the FX framework uses the function name in the log message